### PR TITLE
Add docs for 3rd-party tools

### LINKED ISSUE #1112: We need a doc for 3rd party tools
Where would be the best place for 3rd party tools like [helm](https://github.com/kubernetes/helm) and [kompose](https://github.com/skippbox/kompose)? @jaredbhatti @pwittrock @kubernetes/docs

### PR DESCRIPTION
**Pull request by janetkuo**
Fixes #1112, preview: http://janetkuo.github.io/docs/user-guide/third-party-tools/

Haven't found a great place for it since we don't have "Tools" section yet (under "Reference" for now perhaps? @jaredbhatti) 

@kubernetes/docs @pwittrock

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/kubernetes.github.io/1120)

### CODE REVIEW

**Comment by pwittrock** [comment]
@devin-donnelly WDYT?

**Comment by jaredbhatti** [comment]
Should third party tools have a top-level nav item?

**Comment by janetkuo** [comment]
> Should third party tools have a top-level nav item?

Yes, or we can have a "Tools" top nav, which include all tools (each as one left nav item), which include `kubectl`, dashboard, client libraries, 3rd party tools, ...

**Comment by devin-donnelly** [comment]
Janet,

I'd go ahead and add "Tools" as a top-level navigation item. You can include the tools in a left nav for that section. I'd structure it like this:

- kubectl
- Dashboard
- Client Libraries
  - Go
  - Java
  - etc.
- Third-Party Tools
  - Helm
  - Kompose

**Comment by janetkuo** [comment]
@devin-donnelly why is this removed from the milestone? Does this mean we won't have this doc or top-level navigation item before 1.4 launch?

**Comment by janetkuo** [comment]
@devin-donnelly I added the "Tools" top nav and add this doc under it

File: docs/tools/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+---
+
+The Tools section of the Kubernetes documentation is a work in progress
+
+#### Third-Party Tools
+
+* [Recommended third-party tools, including Helm and Kompose](/docs/tasks/third-party-tools)
```
Comment (devin-donnelly) [inline docs/tools/index.md]: Link 404s. Needs to go to /docs/tools/third-party-tools

**Comment by janetkuo** [comment]
@devin-donnelly doc updated; PTAL

**Comment by devin-donnelly** [comment]
LGTM. Thanks a lot, Janet!

**Pull request merged**